tests/Jellyfin.Server.Implementations.Tests/Test Data/Updates/manifest-stable.json

```diff
@@ -13,15 +13,17 @@
                 "targetAbi": "10.6.0.0",
                 "sourceUrl": "https://repo.jellyfin.org/releases/plugin/anime/anime_10.0.0.0.zip",
                 "checksum": "93e969adeba1050423fc8817ed3c36f8",
-                "timestamp": "2020-08-17T01:41:13Z"
+                "timestamp": "2020-08-17T01:41:13Z",
+                "assemblies": [ "Jellyfin.Plugin.Anime.dll" ]
```

Comment:
Why would the assemblies information need to be present in the repo manifest?
Would it not be more appropriate to only include it in the plugin metadata (`meta.json`)?

The content of the items in the versions array and the `meta.json` file in the plugin archive are not exact duplicates.

```json
{
    "category": "Metadata",
    "changelog": "### Bug Fixes ###\n- fix client string (#19) @Shadowghost\n\n### Code or Repo Maintenance ###\n- Fix readme (#20) @Shadowghost",
    "description": "This plugin provides images from the Cover Art Archive https://musicbrainz.org/doc/Cover_Art_Archive and depends on the MusicBrainz metadata provider to know what images belong where\n",
    "guid": "8119f3c6-cfc2-4d9c-a0ba-028f1d93e526",
    "imageUrl": "https://repo.jellyfin.org/releases/plugin/images/jellyfin-plugin-coverartarchive.png",
    "name": "Cover Art Archive",
    "overview": "MusicBrainz Cover Art Archive",
    "owner": "jellyfin",
    "targetAbi": "10.8.0.0",
    "timestamp": "2022-01-08T17:22:30Z",
    "version": "7.0.0.0"
}
```

Further I'd like to note that the Anime plugin is deprecated, and as such might not be the best test target (it was split into separate plugins)